include/clang/Sema/DeclSpec.h

```diff
@@ -1287,11 +1287,18 @@ struct DeclaratorChunk {
     /// \brief The end location of the exception specification, if any.
     unsigned ExceptionSpecLocEnd;
 
+    /// The location of the ':' for the return bounds expression in the source
+    unsigned ReturnBoundsColon;
```

Comment:
>ReturnBoundsColon [](start = 13, length = 17)

[Nit] I'd suggest including 'Loc' in the name to make it clear that this is a location.